src/model/template.paths.ts

```diff
@@ -2,3 +2,5 @@ export const INDEX: string = "index";
 export const COMPANY_NUMBER: string = "company-number";
 export const CHECK_COMPANY: string = "check-company";
 export const STILL_REQUIRED: string = "still-required";
+export const CONFIRMATION_STILL_REQUIRED: string = "confirmation-still-required";
```

Comment:
So yes, see comment above. I think these constants should refer to 'company' and not 'confirmation'.

Reply:
They need to refer to 'confirmation'